src/presentation/http/http-api.ts

```diff
@@ -186,6 +189,18 @@ export default class HttpApi implements Api {
     });
   }
 
+  /**
+   * Adds support for multipart requests
```

Comment:
is it possible to add multipart support only for "/upload" router?

Reply:
I think, that i can only register plugin before initializing route